simple-cipher/simple-cipher.js

```diff
@@ -0,0 +1,54 @@
+const alphabet = "abcdefghijklmnopqrstuvwxyz".split('');
+
+function randomNumberFromMinToMax(min, max) {
+  return Math.floor(Math.random() * (max - min + 1) + min)   
+}
+
+function randomLetter() {
+  let shift = 9;
+  let number = randomNumberFromMinToMax(1, 25) + shift;
+  return number.toString(36);
+}
+
+function randomKey(length = 100, list = []) {
+  debugger;
+  if (length === list.length) {
+    return list.join('');
+  }
+  return randomKey(length, [...list, randomLetter()]);
```

Comment:
Think about the case case when you'd like to use randomKey to generate key based on e.g. numbers or some other random stuff.